app/main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
you should add skills to previous skillls in init, not write them manually in each class

Reply:
i cant understand how i can do this

Reply:
use `self.skills += ["JavaScript", "HTML", "CSS"]` in `_init__`

Reply:
do it everywhere